README.md

```diff
@@ -1,9 +1,24 @@
 # ironArray Community Edition
 
+## What it is
+
 ironArray Community is the open source version of [ironArray](https://ironarray.io).  It has been developed to mimic the same API than h5py or Zarr.  It implements the support for simple and double floating-point data using a metalayer. With the community edition of ironArray you can extract slices from floating-point datasets in a simple way!
 
 For more info, see the docs at: https://ironarray.io/docs/iarray-community/
 
-Have fun!
+
+## Installing
+
+ironArray Community is now offering a source and a Python wheel for any platform.
```

Comment:
Suggestion: ironArray Community is a pure Python package, so it can easily be installed for any platform.